spec/services/solidus_bolt/accounts/detail_service_spec.rb

```diff
@@ -18,9 +18,10 @@
       let(:access_token) { ENV['BOLT_ACCESS_TOKEN'] }
 
       it 'receives a successful response' do
-        expect(api.call).to match hash_including('profile')
-        expect(api.call).to match hash_including('addresses')
-        expect(api.call).to match hash_including('payment_methods')
+        api_call = api.call
+        expect(api_call).to match hash_including('profile')
+        expect(api_call).to match hash_including('addresses')
+        expect(api_call).to match hash_including('payment_methods')
```

Comment:
Nice catch.
Why didn't you considered to change the subject?

Reply:
oversight

fixed in eb21f69588740a8520b490952797075f6bfbd56d